telco-hub/configuration/reference-crs/required/gitops/app-project.yaml

```diff
@@ -8,6 +8,15 @@ spec:
   destinations:
     - namespace: '*'
       server: '*'
+  namespaceResourceWhitelist:
+    - group: 'argoproj.io'
+      kind: Application
+    - group: 'argoproj.io'
+      kind: AppProject
+    - group: 'operators.coreos.com'
+      kind: OperatorGroup
+    - group: 'operators.coreos.com'
+      kind: Subscription
```

Comment:
This policy shouldn't need these resource types. Was this needed in your env?

Reply:
I gave it another spin in my environment, and they weren't needed indeed.